packages/picasso.js/src/core/chart-components/ref-line/__tests__/refline.spec.js

```diff
@@ -230,6 +234,99 @@ describe('reference lines', () => {
     ]);
   });
 
+  it('should render basic line with RTL label on X with scale when scale.min = scale.max', () => {
```

Comment:
Remove the "RTL" part of this test. Including the RTL in the config. It adds no value to the test of this logic.